src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -233,5 +233,10 @@ public override int GetHashCode()
         #endregion // Equality
 
         public abstract void Visit(ITypeVisitor vistor);
+
+        internal virtual string DefaultExpressionValue()
```

Comment:
>internal virtual string DefaultExpressionValue() [](http://example.com/codeflow?start=8&length=48)

we need this as a form of getting inline types in an expression. 
These expressions may eventually get superceded by the inline type work.